pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -44,41 +44,64 @@ func SetDefaults_ControllerManagerConfiguration(obj *ControllerManagerConfigurat
 	if obj.Controllers.Bastion == nil {
 		obj.Controllers.Bastion = &BastionControllerConfiguration{}
 	}
-	if obj.Controllers.Bastion.ConcurrentSyncs == 0 {
-		obj.Controllers.Bastion.ConcurrentSyncs = 5
+	if obj.Controllers.Bastion.ConcurrentSyncs == nil {
+		v := 5
```

Comment:
What about having a variable `defaultConcurrentSyncs = 5` which you can reuse everywhere instead of redefining `v := 5` everywhere?

Reply:
https://github.com/gardener/gardener/blob/7d52217e6b92881dba08aabf3432048e7e6e08d9/pkg/gardenlet/apis/config/v1alpha1/types.go#L704-L705

Should we define it in as a  types like gardenlet?